deploy/terraform-custom-datacommons/README.md

```diff
@@ -0,0 +1,197 @@
+# Custom Data Commons Terraform Deployment
+
+## Overview
+
+Deploying your own Custom Data Commons instance on Google Cloud Platform (GCP) lets you to host and manage your own data while leveraging [Google's Data Commons](https://datacommons.org/) repository of publicly available information. This deployment is managed using Terraform, which automates the provisioning of infrastructure and services on GCP.
+
+## Features
+
+* Creates Data Commons Website service container in Cloud Run
+* Creates Data Commons Data task container in Cloud Run
+* Enables all required Google Cloud APIs
+* Creates Redis instance (optional)
+* Creates MySQL instance
+* Creates new service account with minimum required permissions
+* Automatically provisions required Data Commons API key. Stores key in GCP secrets container.
+* Automatically provisions required Data Commons API key. Stores key in GCP secrets container.
```

Comment:
Should these lines be removed since DC api key is an input to the process and has to be requested separately?

Reply:
Fixed